benchmark/single-source/Breadcrumbs.swift

```diff
@@ -29,6 +29,7 @@ public let benchmarks: [BenchmarkInfo] = [
 
   CopyUTF16CodeUnits(workload: asciiWorkload, count: 500).info,
   CopyUTF16CodeUnits(workload: mixedWorkload, count: 500).info,
+  CopyUTF16CodeUnits(workload: longMixedWorkload, count: 50).info,
 
```

Comment:
The speedup on `_nativeCopy` shows up clearly on this benchmark but not on the ones above it, which is interesting